_posts/2019-11-09-using-vault-with-strimzi.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+title:  "Using Vault PKI secrets engine as a CA with the Strimzi Kafka Operator"
+date: 2018-11-14
+author: chris_vest
+---
+
+<!--more-->
+
+# A brief introduction to Vault
+
+If you're here you probably know what [Vault](https://www.vaultproject.io/) is and why it's useful; but for the uninitiated here is a brief description from the creators themselves. The Hashicorp Vault project "secures, stores, and tightly controls access to tokens, passwords, certificates, API keys, and other secrets in modern computing [...]".
+
+So how does Vault fit in with Strimzi and Kafka? Well, Vault has the concept of [`Secrets Engines`](https://www.vaultproject.io/docs/secrets/):
+
+> Secrets engines are components which store, generate, or encrypt data. Secrets engines are incredibly flexible, so it is easiest to think about them in terms of their function. Secrets engines are provided some set of data, they take some action on that data, and they return a result.\
+> Some secrets engines simply store and read data - like encrypted Redis/Memcached. Other secrets engines connect to other services and generate dynamic credentials on demand. Other secrets engines provide encryption as a service, totp generation, certificates, and much more.
```

Comment:
"totp" ? it's a typo I guess ...

Reply:
It's actually a reference to "Time-based One-time Password algorithm"! I'll add a note.

Reply:
ah!! I didn't get it even because it wasn't quoted or something like this. I think that a note could be better, thanks!